lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -259,41 +237,29 @@ class DeviceInfoRequestCommand extends BCNLCommand {
   }
 
   // Override send(), because we do a 'double read' after sending the request
-  send(hidDevice, readTimeout = BCNLMessage.READ_TIMEOUT_MS,
+  async send(hidDevice, readTimeout = BCNLMessage.READ_TIMEOUT_MS,
     sendDelay = BCNLMessage.SEND_DELAY_MS) {
-    // eslint-disable-next-line no-unused-vars
-    return new Promise((resolve, reject) => {
-      let response1 = null;
-
-      // Timeout might be zero, but the callback will fire anyway
-      // We use sendMessage instead of super.send() because we want to pull raw
-      // data only, and we can't determine response types until after we've checked
-      // them. The CNL can send the messages in different orders.
-      this.sendMessage(hidDevice)
-        .then(() => BCNLMessage.wait(sendDelay))
-        .then(() => this.readMessage(hidDevice, readTimeout))
-        .then((response) => {
-          response1 = response;
-        })
-        .then(() => BCNLMessage.wait(sendDelay))
-        .then(() => this.readMessage(hidDevice, readTimeout))
-        .then((response2) => {
-          let astmInfo = '';
-
-          if (response1[0] === BCNLMessage.ASCII_CONTROL.EOT) {
-            astmInfo = Buffer.from(response1).toString('ascii');
-            new BCNLCommandResponse(response2).checkAsciiControl(BCNLMessage.ASCII_CONTROL.ENQ);
-          } else {
-            astmInfo = Buffer.from(response2).toString('ascii');
-            new BCNLCommandResponse(response1).checkAsciiControl(BCNLMessage.ASCII_CONTROL.ENQ);
-          }
-          const message = new DeviceInfoRequestResponse(astmInfo);
-          resolve(message);
-        })
-        .catch((err) => {
-          reject(err);
-        });
-    });
+    // Timeout might be zero, but the callback will fire anyway
+    // We use sendMessage instead of super.send() because we want to pull raw
+    // data only, and we can't determine response types until after we've checked
+    // them. The CNL can send the messages in different orders.
+    await this.sendMessage(hidDevice);
+    await Timer.wait(sendDelay);
```

Comment:
It *probably* isn't. In fact, I only put them in here because it was in the CLP code. I'll remove them for now and see if we get any adverse affects.